### DISCUSSION TITLE
Sound PTU Flaps

### DISCUSSION THREAD

**Discussion by six75** [flybywiresim/aircraft#3230] [Q&A] [answered]
Hello and congratulations to all for this masterpiece you are creating, 
I would like to have the sounds in the PTU cabin again when starting the engines and flaps when they extend, 
or the option to activate or deactivate the sounds
thanks

**Comment by hotshotp** [comment, marked as answer]
Flaps and PTU are heard in the cabin like you said

**Reply by six75** [reply]
I downloaded the custom FBW and Dev version but I don't hear the PTU and flaps sounds from inside the VC
the flaps I don't mean the lever that you hear, but the sound of the flaps extending

**Reply by hotshotp** [reply]
> I downloaded the custom FBW and Dev version but I don't hear the PTU and flaps sounds from inside the VC
> the flaps I don't mean the lever that you hear, but the sound of the flaps extending

Oh well you said cabin not cockpit in your post. It isn't realistic and we won't be adding the option until we get the EFB which isn't for a while

**Reply by six75** [reply]
ok from the window view the flaps can be heard, 
while the PTU is not! it's correct?

**Reply by hotshotp** [reply]
no, the ptu can be heard in the window view after you start one engine
